Bluejay/Bluejay/ConnectionOptions.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import CoreBluetooth
+
+/// Allow configuring whether iOS can display a system alert when certain conditions are met while your app is suspended, usually an alert dialog outside of your app in the Home screen for example.
```

Comment:
Comment seems a little overly specific. Even though the only per connection options right now ARE for whether to show the system alert, the struct IN THEORY could specify other per-connection operations, either stuff added to CB later, or something of our own (like the suggestion above to put timeout in this, even though, as I said I'm not sure that's actually a good idea in practice). I think the documentation for the struct itself should just describe the overall purpose of the options, rather than focusing on the specific options it has now.

Reply:
Change to this comment will depend on whether we decide to rename this struct, but either way, yes, I think we can and should focus more on the overall purpose. Once we have a decision, I will improve the comment here as discussed.